internal/iam/auth.go

```diff
@@ -2,9 +2,11 @@ package iam
 
 // Credentials holds the credentials for accessing Sauce Labs.
 type Credentials struct {
-	Username  string `yaml:"username"`
-	AccessKey string `yaml:"accessKey"`
-	Source    string `yaml:"-"`
+	Username  string        `yaml:"username"`
+	AccessKey string        `yaml:"accessKey"`
+	Region    string        `yaml:"region,omitempty"`
+	Regional  []Credentials `yaml:"regional,omitempty"`
```

Comment:
As was noted in the jira ticket, the name `Regional` was purely for inspiration. I don't think it represents idiomatic go. Would be nice to have some sort of a plural here. `Regions` might be confusing w/r/t/ to the `Region` string, since there's a type change between the two. `Regionals` might be a nice alternative to its longer sibling `RegionalCredentials`.

@mhan83 thoughts? alternatives?

Reply:
Hmmm, I'm worried my PR for custom regions makes this even more confusing. I wonder if its more natural for regional credentials to be part of region configuration. Like if `regions.yml` looked like this?:

```yaml
- name: us-west-1
  credentials:
    username: foo
    accessKey: bar
- name: local
  credentials:
    username: foo
    accessKey: bar
  endpoints:
    apiBaseURL: bloo
    appBaseURL: blah
    webdriverBaseURL: blee
```